src/vast_tracker.js

```diff
@@ -339,6 +371,10 @@ export class VASTTracker extends EventEmitter {
    */
   trackImpression(macros = {}) {
     if (typeof macros !== 'object') {
+      this.emit('TRACKER-error', {
+        message: 'trackImpression parameter has the wrong type',
```

Comment:
The message can be more explicit in this case and next ones below
```suggestion
        message: 'trackImpression given macros has the wrong type',
```